src/sentry/mediators/external_requests/util.py

```diff
@@ -52,14 +52,26 @@ def send_and_save_sentry_app_request(url, sentry_app, org_id, event, **kwargs):
 
     buffer = SentryAppWebhookRequestsBuffer(sentry_app)
 
+    slug = sentry_app.datadog_name
+
     try:
         resp = safe_urlopen(url=url, **kwargs)
     except RequestException:
+        metrics.incr(
```

Comment:
I'd refactor this out into it's own method like we did with [track_response_code](https://github.com/getsentry/sentry/pull/17060/commits/256338bbf6b85eb88e3f6add553dafe91d6909ea). That way, we only mention `integration-platform.http_response` once. And that makes it much less likely that in the future, we have metrics going out under two different names.